guides/common/modules/ref_system-requirements.adoc

```diff
@@ -41,6 +41,10 @@ endif::[]
 * A system umask of 0022
 * Full forward and reverse DNS resolution using a fully-qualified domain name
 
+NOTE: The shortnames in the hostnames are no longer supported for the {Project} and {SmartProxy} server.
```

Comment:
I think it should be "Project does not support using shortnames" instead of "no longer", because we deliver versioned documentation. This means that users always get the appropriate docs for their version of Foreman.

cc @melcorr 

For the note box, I am fine with it, but maybe you can reword it as a requirements rather then "warning".

Reply:
At the rate people are trying to add notes to our docs, we would have only notes. 
It reads to me like a regular sentence. Let's remove the NOTE prefix. 
I agree with removing the no longer for the reasons @maximiliankolb outlined.

Reply:
Noted 👍

Reply:
Modified.